backend/tournesol/views/polls.py

```diff
@@ -201,3 +202,16 @@ def get_queryset(self):
         queryset = self.annotate_with_total_score(queryset, self.request, poll)
         queryset = self.filter_unsafe(queryset, filters)
         return queryset.order_by("-total_score", "-pk")
+
+
+class PollsCriteraScoreDistributionView(PollScopedViewMixin, RetrieveAPIView):
+    poll_parameter = "name"
+
+    permission_classes = []
+    queryset = Entity.objects.none()
+    serializer_class = EntityCriteraDistributionSerializer
+
+    def get_object(self):
+        """ Get object based on the entity uid """
+        entity_uid = self.kwargs.get("uid")
+        return Entity.objects.get(uid=entity_uid)
```

Comment:
This raises a 500 (see my suggestion in the tests). Let's use `get_object_or_404`?

Reply:
Thanks it's fixed and your suggestion is added